### DISCUSSION TITLE
How can I disable the media overlay?

### DISCUSSION THREAD

**Discussion by crystalpoi** [pear-devs/pear-desktop#2717] [Q&A] [answered]
Hi. I just started to use the app. I liked it a lot but I can't find a way to disable the media overlay which appears all the time when I pause or resume a song? That affects my FPS while playing games.

**Comment by Lockszmith-GH** [comment]
Are you referring to the notifications?
There is a plugin called Notifications you can disable.
Also, on Windows (and I would assume other OSes as well), notifications can be disabled on the OS level.

**Reply by crystalpoi** [reply]
The situation I'm talking about is not the notifications plugin. It displays a notification when a song starts. What I'm talking about is that when I pause or resume a song using my keyboard, an overlay appears on Windows and I can't disable that. I tried the notification plugin thinking it might be related, but it didn't work. The application does not appear in the Windows notifications section. This is not a notification problem, but rather a media overlay issue. We can turn it off in browsers, but I couldn't find it in this application.

**Reply by JellyBrick** [reply, marked as answer]
plugins - shortcuts - Override MediaKeys
